### PR TITLE
upgrade ci

### PR DESCRIPTION
**Pull request by k0nserv**
Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/main/Example/Backpack%20Screenshot/Screenshots.swift)
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

**Review by marianeum — APPROVED** [review]

**Review by olliecurtis — APPROVED** [review]

File: Example/SnapshotTests/BPKSnapshotTest.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /*
  *  Copyright (c) 2017-2018, Uber Technologies, Inc.
  *  Copyright (c) 2015-2018, Facebook, Inc.
+ *  Copyright (c) 2018-2021, Skyscanner Ltd.
```
Comment (k0nserv) [inline Example/SnapshotTests/BPKSnapshotTest.h:4]: @georgegillams why doesn't this file have our typical header? Did we copy some of this code from `FBSnapshotTestCase`?

**Comment by olliecurtis** [comment]
@k0nserv looking at the CI failure it seems according to https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#installed-sdks iphonesimulator 14.4 is not available for xcode 12.5 so we need to bump  https://github.com/Skyscanner/backpack-ios/blob/upgrade-ci/.github/workflows/ci.yml#L131 up to `14.5`

**Comment by k0nserv** [comment]
I thought I had already done that 🤔

**Comment by k0nserv** [comment]
Ah, I had missed that place it seems.

**Pull request merged**